src/Request.php

```diff
@@ -16,11 +16,11 @@
     use RequestTrait;
 
     /**
-     * @param string $method HTTP method
-     * @param string|UriInterface $uri URI
-     * @param array $headers Request headers
-     * @param string|null|resource|StreamInterface $body Request body
-     * @param string $version Protocol version
+     * @param string                               $method  HTTP method
```

Comment:
This won't be parse-able via IDEs. At least PhpStorm.

Reply:
Yeah, I saw that. I will figure out what php-cs-fixer rule that disables this.